bench/app.py

```diff
@@ -201,7 +201,10 @@ def remove(self, no_backup: bool = False):
 		active_app_path = os.path.join("apps", self.name)
 
 		if no_backup:
-			shutil.rmtree(active_app_path)
+			if not os.path.islink(active_app_path):
+				shutil.rmtree(active_app_path)
+			else:
+				os.remove(active_app_path)
```

Comment:
Errrm... os.remove doesn't work with folders? 

check this: https://docs.python.org/3/library/os.html#os.remove

Reply:
My bad. This makes sense 👍